src/main/scala/gpp/ui/forms/EnumViewMultipleSelect.scala

```diff
@@ -0,0 +1,217 @@
+  label:                js.UndefOr[ShorthandS[String]] = js.undefined,
+  labeled:              js.UndefOr[Boolean] = js.undefined,
+  loading:              js.UndefOr[Boolean] = js.undefined,
+  minCharacters:        js.UndefOr[JsNumber] = js.undefined,
+  noResultsMessage:     js.UndefOr[ShorthandS[VdomNode]] = js.undefined,
+  onBlur:               js.UndefOr[Callback] = js.undefined,
+  onBlurE:              js.UndefOr[OnBlur] = js.undefined,
+  onClick:              js.UndefOr[Callback] = js.undefined,
+  onClickE:             js.UndefOr[OnClick] = js.undefined,
+  onChange:             js.UndefOr[FormDropdown.OnChange] = js.undefined,
+  onChangeE:            js.UndefOr[FormDropdown.OnChangeE] = js.undefined,
+  onClose:              js.UndefOr[Callback] = js.undefined,
+  onCloseE:             js.UndefOr[OnClose] = js.undefined,
+  onFocus:              js.UndefOr[Callback] = js.undefined,
+  onFocusE:             js.UndefOr[OnFocus] = js.undefined,
+  onLabelClick:         js.UndefOr[Callback] = js.undefined,
+  onLabelClickE:        js.UndefOr[OnLabelClick] = js.undefined,
+  onMouseDown:          js.UndefOr[Callback] = js.undefined,
+  onMouseDownE:         js.UndefOr[OnMouseDown] = js.undefined,
+  onOpen:               js.UndefOr[Callback] = js.undefined,
+  onOpenE:              js.UndefOr[OnOpen] = js.undefined,
+  onSearchChange:       js.UndefOr[OnSearchChange] = js.undefined,
+  onSearchChangeE:      js.UndefOr[OnSearchChangeE] = js.undefined,
+  open:                 js.UndefOr[Boolean] = js.undefined,
+  openOnFocus:          js.UndefOr[Boolean] = js.undefined,
+  placeholder:          js.UndefOr[String] = js.undefined,
+  pointing:             js.UndefOr[Pointing] = js.undefined,
+  renderLabel:          js.UndefOr[RenderLabel] = js.undefined,
+  required:             js.UndefOr[Boolean] = js.undefined,
+  scrolling:            js.UndefOr[Boolean] = js.undefined,
+  search:               js.UndefOr[Boolean | SearchFunction] = js.undefined,
+  searchInput:          js.UndefOr[ShorthandS[VdomNode]] = js.undefined,
+  searchQuery:          js.UndefOr[String] = js.undefined,
+  selectOnBlur:         js.UndefOr[Boolean] = js.undefined,
+  selectOnNavigation:   js.UndefOr[Boolean] = js.undefined,
+  selectedLabel:        js.UndefOr[JsNumber | String] = js.undefined,
+  simple:               js.UndefOr[Boolean] = js.undefined,
+  tabIndex:             js.UndefOr[String | JsNumber] = js.undefined,
+  text:                 js.UndefOr[String] = js.undefined,
+  tpe:                  js.UndefOr[String] = js.undefined,
+  trigger:              js.UndefOr[VdomNode] = js.undefined,
+  upward:               js.UndefOr[Boolean] = js.undefined,
+  width:                js.UndefOr[SemanticWidth] = js.undefined,
+  wrapSelection:        js.UndefOr[Boolean] = js.undefined,
+  modifiers:            Seq[TagMod] = Seq.empty
+)(implicit
+  val enum:             Enumerated[A],
+  val show:             Show[A],
+  val effect:           Effect[F]
```

Comment:
Why we need `Effect`? Can we get by with a weaker restriction?

Reply:
Again, just copied from EnumViewSelect, I'll see what the minimum requirement is.

Reply:
This comes all the way from `crystal`, which ends up doing an `Effect[F].runAsync` to execute the code in a `Callback`.

BTW, for a future PR, I realize now that we shouldn't rely on `Show` to show the values in the UI.

Reply:
There is a `Display` typeclass in Enumerated.scala. Should we be using that, instead of `Show`? I can update the typeclass instances in `explore`.

Reply:
We are not really using `Display` anywhere but we may need to come up with something like that.

Reply:
It seems like this is one of the things that Display was meant for. Do we want to use it? Or make something else? Maybe it shouldn't be in gem.util, though?

Reply:
How about using the `Format` optic? IIRC, the idea is to use `Format` to parse and serialize. Are there cases where we want to show a value to the user in a different format than serialized to DB/JSON/etc?

Reply:
Format is a good candidate, I wonder if we need to be implicit as there maybe many Format instances for a given type

Reply:
Agreed that it needs to be explicit. We actually do have an `InputFormat` to format to/from `String`, which we use in `InputEV`, maybe we should unify the mechanism.

Reply:
I would think there would be many cases where the DB/wire format would be different from what we want to show the user. For example, for the Enumerated values we want to save the tag to the DB, but show the label to the user. I've looked at Format a bit, but will need to think about that. I'll take a look at InputFormat, too.

Reply:
`Format` doesn't seem like an appropriate candidate for displaying the text in a select control. Format is bidirectional, and we never want to go from display format (`label` for most of the Enumerated instances) to the the Enumerated itself. My understanding is that the `tag` is the only thing about an Enumerated that is considered stable.

To generalize the select control more, we could use a `Format` for converting back and forth between the `tag` and the Enumerated when creating the `value`s for the option list and converting a selection back to the Enumerated. But, I think in this instance, we could actually use a Prism since there won't be anything to normalize - there is only one valid tag. In fact, since we create the option list, it should only create valid tags, so we could even use an `Iso` here. But, since that isn't true in general for going from string to Enumerated, it wouldn't be a valid `Iso` instance.

Reply:
You are right, in this case what we want to show to the user is not the same as what we want to use as encoding. So I definitely agree on using a different typeclass for that. `Display` seems to have been created for this purpose so we may as well go with it.

For `Enumerated`<->`String` conversion, I also agree that we should use the tightest optic a possible and `Prism` seems to be all that the `EnumSelect`s need. Maybe it's practical to have a convenience `InputPrism[A]` that fixes the other paremter to a `String`, just like we have for `InputFormat`?

At some point we may want to create `Enumerated`s from alternate `String` representations (eg: lowercase, snakecase); but this is not what `EnumSelect`s will do, so let's cross that bridge if we ever come to it.

Reply:
I modified the Display typeclass in gemini-hlsw/lucuma-core#149. When that is merged and used in this project, I can modify the EnumView*Selects to make use of that. I can also make Display instances for the Enumerateds in explore that currently have Show instances.